hydrogen.py

```diff
@@ -335,22 +335,22 @@ def load(self, requirements_file=None):
             requirements_file = self.filename
             if requirements_file is None:
                 raise ValueError("no filename provided")
-        elif isinstance(requirements_file, basestring):
-            requirements_file = Path(basestring)
+        elif isinstance(requirements_file, text_type):
+            requirements_file = Path(requirements_file)
```

Comment:
I am not sure about this line. The previous incarnation doesn't seem like it would have worked, so I did what I think you meant to do here, but let me know if this needs to change.